lua/lspconfig/lemminx.lua

```diff
@@ -0,0 +1,43 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local name = "lemminx"
+local bin_name = "lemminx"
+
+configs[name] = {
+  default_config = {
+    cmd = {
+      bin_name
+    },
+    filetypes = {"xml", "xsd", "svg"};
+    root_dir = function(filename)
+      return util.root_pattern(".git")(filename) or util.path.dirname(filename)
+    end;
+  };
+  docs = {
+    description = [[
+https://github.com/eclipse/lemminx
+
+Features:
```

Comment:
I would match the documentation style of other plugins, see elixirls for the standard "We don't modify your path blah blah" bit I put in. I also wouldn't recommend listing the capabilities as these may become out of date.

Reply:
fixed